curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/64649b108df035051cb2ba2d.md

```diff
@@ -13,16 +13,14 @@ And with those changes, your freeCodeCamp forum leaderboard project is now compl
 
 # --hints--
 
-Your opening anchor tag should have a `target` attribute set to `_blank`.
 
-```js
-assert.match(code, /<a\s+(?:class=['"]post-title['"]\s+target=['"]_blank['"]|target=['"]_blank['"]\s+class=['"]post-title['"])(?:.|\n)*?<\/a>/);
-```
```

Comment:
We should have a test for just checking for the target blank. 

It is important to keep the tests granular in nature so campers will know where they went wrong.
If you just have one giant test, then it will be harder for them to know where the issue is

You will need to add this test back in and make sure there are no conflicts with the other test